cmd/podman/play_kube.go

```diff
@@ -243,6 +244,14 @@ func kubeContainerToCreateConfig(containerYAML v1.Container, runtime *libpod.Run
 	containerConfig.Name = containerYAML.Name
 	containerConfig.Tty = containerYAML.TTY
 	containerConfig.WorkDir = containerYAML.WorkingDir
+
+	imageData, _ := newImage.Inspect(ctx)
+
+	containerConfig.User = "0"
```

Comment:
Is this OK in a rootless scenario?  But having said that, I'm not sure what else to set it to.

Reply:
rootless is actually disabled in play kube right now. This is also the default in pkg/spec/createconfig.go 's ParseCreateConfig. So I think it's safe?

Reply:
Yeah, rootless uses a user namespace to map UIDs, so we want to maintain the UID 0 default there too